buurbak/src/app/Components/Footer.tsx

```diff
@@ -0,0 +1,130 @@
+"use client";
+
+import { ArchiveBoxXMarkIcon } from "@heroicons/react/24/outline";
+import Image from "next/image";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+import Logo from "../Assets/buurbakLogoBigColor.svg";
```

Comment:
Zet een iets langere beschrijving hierbij zodat je niet op Logo en Logom uitkomt. Denk aan LogoBC en LogoMC of iets dergelijks

Reply:
Yess ga ik aanpassen goed idee